src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,59 @@
+import VenueChat from "components/molecules/VenueChat";
+import ChatsList from "components/molecules/ChatsList";
+import LiveSchedule from "components/molecules/LiveSchedule";
+import { useSelector } from "hooks/useSelector";
+import { useVenueId } from "hooks/useVenueId";
+import React, { useState } from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+import "./Sidebar.scss";
+
+const Sidebar = () => {
+  const venueId = useVenueId();
+  useFirestoreConnect({
+    collection: "users",
+    where: ["enteredVenueIds", "array-contains", venueId],
+    storeAs: "chatUsers",
+  });
+  const [tab, setTab] = useState(0);
+  const { privateChats, chatUsers } = useSelector((state) => ({
+    privateChats: state.firestore.ordered.privatechats,
+    chatUsers: state.firestore.data.chatUsers,
+  }));
+  const isEnabled = chatUsers && privateChats;
+  return (
+    <div className="sidebar-container">
+      <div className="sidebar-slide-btn">
+        <div className="slide-btn-arrow-icon"></div>
+        <div className="slide-btn-chat-icon"></div>
+      </div>
+      <div className="sidebar-tabs">
+        <div
+          className={`sidebar-tab sidebar-tab_chat ${tab === 0 && "active"}`}
+          onClick={() => isEnabled && setTab(0)}
+        >
+          Party Chat
+        </div>
+        <div
+          className={`sidebar-tab sidebar-tab_private ${tab === 1 && "active"}`}
+          onClick={() => isEnabled && setTab(1)}
```

Comment:
As above RE: `useCallback`

Reply:
this is still an issue that will cause re-rendering due to the wrapper functions (`() =>`) reference changing on every render

Fix:

Close around the state + `useCallback`:
```typescript
const selectMessagesTab =  useCallback(() => {
    isEnabled && setTab(1)
  }, [isEnabled]);
```

Then use that here:
```suggestion
          onClick={selectMessagesTab}
```

Reply:
fixed in 70e8253242c4c56f3678a5035e0e6566912da044